shell/platform/embedder/embedder.cc

```diff
@@ -2998,17 +2998,12 @@ FlutterEngineResult FlutterEngineNotifyDisplayUpdate(
     case kFlutterEngineDisplaysUpdateTypeStartup: {
       std::vector<std::unique_ptr<flutter::Display>> displays;
       for (size_t i = 0; i < display_count; i++) {
-        if (embedder_displays[i].single_display) {
-          displays.push_back(std::make_unique<flutter::Display>(
-              embedder_displays[i].refresh_rate));
-        } else {
-          displays.push_back(std::make_unique<flutter::Display>(
-              embedder_displays[i].display_id,
-              embedder_displays[i].refresh_rate));
-        }
+        displays.push_back(std::make_unique<flutter::Display>(
+            embedder_displays[i].display_id, embedder_displays[i].refresh_rate,
```

Comment:
FYI, this should use the `struct_size` to iterate through the array dynamically: https://github.com/flutter/flutter/issues/121352

I don't think it's strictly necessary for this change as you haven't added any new members to `FlutterEngineDisplay`... but please double-check me on that.

Reply:
No changes to FlutterEngineDisplay yet. That'll be in the resolution to https://github.com/flutter/flutter/issues/125939

Reply:
Sounds good, marking this as resolved